@stellar/design-system/src/components/Typography/index.tsx

```diff
@@ -0,0 +1,145 @@
+import "./styles.scss";
+
+// =============================================================================
+// Heading
+// =============================================================================
+interface HeadingProps extends React.HtmlHTMLAttributes<HTMLHeadingElement> {
+  as: "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
+  size: "xxl" | "xl" | "lg" | "md" | "sm" | "xs";
```

Comment:
I suggest we use string values instead of enums as much as possible. I personally find it more convenient (VS Code has good hints for that, and it's more clear than enum + no need to import anything extra). I'm open to feedback. If you feel strongly against it, please comment. We'll go with what is the most acceptable for all. :slightly_smiling_face: 

![image](https://user-images.githubusercontent.com/7346473/216686018-e4896211-77c5-408f-bc37-d658d456a800.png)